internal/model/plugin.go

```diff
@@ -19,8 +19,27 @@ type Plugin struct {
 	Manifest        *mattermostModel.Manifest `json:"manifest"`
 	Enterprise      bool                      `json:"enterprise"` // Indicated if the plugin is an enterprise plugin
 	UpdatedAt       time.Time                 `json:"updated_at"` // The point in time this release of the plugin was added to the Plugin Marketplace
+	PlatformBundles PlatformBundles           `json:"platform_bundles"`
```

Comment:
I'm would like 2/5 to see `UpdatedAt` being kept as the last field just to reduce the diff